configs/client_cli/config.json

```diff
@@ -1,13 +1,13 @@
 {
-  "TORII_API_URL": "127.0.0.1:8080",
+  "TORII_API_URL": "http://127.0.0.1:8080",
   "ACCOUNT_ID": {
-    "name": "root",
-    "domain_name": "global"
+    "name": "alice",
+    "domain_name": "wonderland"
   },
   "PUBLIC_KEY": "ed01207233bfc89dcbd68c19fde6ce6158225298ec1131b6a130d1aeb454c1ab5183c0",
```

Comment:
is public key in multihash format?

Reply:
Yes.

Reply:
could you mention it somewhere in the doc?

Reply:
I will add it to the `contributing` PR.